docs/documentation/upgrading/topics/keycloak/changes-23_0_0.adoc

```diff
@@ -135,3 +135,6 @@ Stream<GroupModel> getTopLevelGroupsStream(RealmModel realm,
 
 Endpoint `GET {keycloak server}/realms/{realm}/groups/{group_id}/children` added as a way to get subgroups of specific groups that support pagination
 
+= Removal of the options to trim the event's length
+
+Since this release Keycloak supports long values for both `EventEntity` details and `AdminEventEntity` representation columns. Therefore it no longer supports options for trimming event detail length `--spi-events-store-jpa-max-detail-length` and `--spi-events-store-jpa-max-field-length`.
```

Comment:
Feels like missing comma to me, but would be great if someone can verify.
```suggestion
Since this release, Keycloak supports long values for both `EventEntity` details and `AdminEventEntity` representation columns. Therefore it no longer supports options for trimming event detail length `--spi-events-store-jpa-max-detail-length` and `--spi-events-store-jpa-max-field-length`.
```

Reply:
Also comma after "Therefore"